src/main/java/com/coveo/configuration/parameterstore/ParameterStorePropertySourceConfigurationProperty.java

```diff
@@ -0,0 +1,19 @@
+package com.coveo.configuration.parameterstore;
+
+public final class ParameterStorePropertySourceConfigurationProperty
```

Comment:
` ParameterStorePropertySourceConfigurationProperties` I feel like this class should have methods that return the values and even hold some logic to know if the lib is enabled for example. Doing this would simplify the post processors class and encapsulate the whole logic for configuration properties in here.